modules/serverless-functions-on-cluster-builds.adoc

```diff
@@ -27,14 +27,14 @@ include::snippets/technology-preview.adoc[leveloffset=+1]
 +
 [source,terminal]
 ----
-$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.22.0/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks.yaml
+$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.24.0/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks.yaml
 ----
```

Comment:
Here we need to include the s2i task as well

```
$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.25.0/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml
```

Reply:
Added, can you please check the wording? https://github.com/openshift/openshift-docs/pull/51030/commits/31b4677cc44b8d6e4bcfdf405b50397bfd8fcd04

Reply:
Actually, use this link instead: https://github.com/openshift/openshift-docs/pull/51030/commits/31b4677cc44b8d6e4bcfdf405b50397bfd8fcd04#diff-3f93bad3a6dccebfacff22389de39b99cf910accf4cadd053414fd97349accd0R40